FabricLogging.py

```diff
@@ -50,10 +52,12 @@ def getLevelName(level):
         return result
     return f"Level {level}"
 
+
 class Filter():
     def __init__(self, name):
         self.name = name
 
+
 class CSVHandler:

```

Comment:
This might be the way, unfortunately, the idea was that if i had so much configuration to do that i needed a config file i would rather try to reduce the complexity. I might include a function that sets config best practice so that developers can use the library without having to do any configuration what-so-ever. still undecided on that.